.copr/Makefile

```diff
@@ -2,7 +2,49 @@ srpm:
 	dnf -y install git
 	git submodule update --init --recursive
 	cd deployment/fedora-package-x64;                    \
-	./create_tarball.sh;                                 \
+    WORKDIR="$( pwd )"; \
```

Comment:
can we update a `./create_tarball.sh` with this instead of embedding it into Makefile? reading it inside a Makefile without syntax highlight and with lots of `\` continuation things is harder than reading a script IMHO...

Reply:
Would reduce duplication also.